import_data/tasks/format_stdout.py

```diff
@@ -24,6 +24,7 @@ def flush(self):
             self.src_stream.write('[{}] {}\n'.format(self.prefix, line))
 
         self.buffer = lines[-1]
+        self.src_stream.flush()
```

Comment:
This finally didn't led to any issue ? I may been confused by [the issue we had with exec.py](https://github.com/QwantResearch/kartotherian_docker/pull/96) :)

Reply:
No, I haven't noticed any issue, and it helps to follow the messages in real-time in our deployed pipeline.